HELM-NOTES.md

```diff
@@ -0,0 +1,196 @@
+# Deploying Apache Hop with Helm on Kubernetes
+
+This document contains technical documentation on how to deploy Apache Hop using Helm on a Kubernetes cluster.
+
+## What is Helm?
+
+Kubernetes can become very complex to manage using standard configuration files (services, pods, volumes, different releases, ... ) which is solved by the use of Helm. Helm is a package manager which can package everything into a simple configurable application. Helm offers easy installing, updating and removal.
+
+> Helm installs charts into Kubernetes, creating a new release for each installation. And to find new charts, you can search Helm chart repositories.
+
+## Prerequisites
+
+The following prerequisites are needed to get started with Apache Hop and Helm.
+
+- A repository which contains a working Hop Docker image
```

Comment:
no need for repository - dockerhub or apache repo should have

Instead of asking people to upload to their own helm repository, we can ask them to use local file system or [github pages](https://medium.com/@mattiaperi/create-a-public-helm-chart-repository-with-github-pages-49b180dbb417) or github release tgz.

we can pick one of github route or find if Apache has any helm repository ( I am sure Apache will have) @hansva can help us.

Reply:
I agree, this should be as easy as possible for the end-user. I spoke with @hansva and he wants to set up an automated build process which pushes images to an official repository.

Maybe leave this as is until the process is running?